packages/engine/Specs/Core/PolygonGeometryLibrarySpec.js

```diff
@@ -0,0 +1,698 @@
+import {
+  ArcType,
+  Cartesian3,
+  Ellipsoid,
+  Math as CesiumMath,
+  PolygonGeometryLibrary,
+} from "../../index.js";
+
+describe("Core/PolygonGeometryLibrary", function () {
```

Comment:
What happens if a user supplies four points to describe a rectangle spanning >180 degrees of longitude? Do we automatically assume min/max are flipped, and treat it as spanning <180 degrees?

Reply:
We assume the shortest path. That would be assuming the min/max are flipped. This is consistant with the rest of the Cesium API. 

You can get around this behavior by supplying additional polygon positions that do not span >180 degrees.